From: Mathieu Trudel-Lapierre <mathieu.trudel-lapierre@canonical.com>
Date: Wed, 2 May 2012 19:35:17 +0200
Subject: Fix the NSS libdir to add a "/nss"

That variable is only used to point to the directory in which nssckbi (the
built-in certificate store of NSS) lives, and it's actually under libdir/nss
rather than directly under libdir.
---
 cmake/modules/FindSMIME.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/modules/FindSMIME.cmake b/cmake/modules/FindSMIME.cmake
index 2537119..ea5be51 100644
--- a/cmake/modules/FindSMIME.cmake
+++ b/cmake/modules/FindSMIME.cmake
@@ -55,7 +55,7 @@ if((WITH_NSPR_INCLUDES STREQUAL "") AND (WITH_NSPR_LIBS STREQUAL "") AND (WITH_N
 		set(MANUAL_NSPR_LIBS "")
 		set(MANUAL_NSS_INCLUDES "")
 		set(MANUAL_NSS_LIBS "")
-		set(MOZILLA_NSS_LIB_DIR "${_nss_libdir}")
+		set(MOZILLA_NSS_LIB_DIR "${_nss_libdir}/nss")
 		return()
 	endif((NOT (mozilla_nspr STREQUAL "")) AND (NOT (mozilla_nss STREQUAL "")))
 endif()
